dxr/plugins/clang/dxr-index.cpp

```diff
@@ -160,7 +160,12 @@ class PreprocThunk : public PPCallbacks {
 #endif
       StringRef searchPath,
       StringRef relativePath,
-      const Module *imported
+#if CLANG_AT_LEAST(19, 0)
+      const Module *suggestedModule,
+      bool moduleImported
+#else
+      const Module *suggestedModule
+#endif
```

Comment:
Could this be
```C++
const Module *suggestedModule
#if CLANG_AT_LEAST(19, 0)
    , bool moduleImported
#endif
```
?

That way we avoid any duplication of `const Module *suggestedModule`.  And it follows the existing pattern that's used for `fileType`.  Both here and at the definition below.